paasta_tools/cli/cmds/spark_run.py

```diff
@@ -530,7 +530,7 @@ def get_docker_image(
 
     docker_url = ""
     if args.image:
-        docker_url = args.image
+        return args.image
```

Comment:
doesn't this mean we'll potentially use an image that doesn't exist?

Reply:
the change to check if the image can be pulled was included recently and has since caused build issues when using a `--image`. for now, we are just trying to roll back the change made, until the ml-compute team can figure out a fix

Reply:
ah, I see!